etna/lib/commands.rb

```diff
@@ -149,16 +150,16 @@ def execute(project_name, execute: false, target_model: 'project', file: "#{proj
 
               if execute
                 puts "Would you like to execute?"
-                if StrongConfirmation.confirm
+                if confirm
                   workflow.synchronize_to_server(@models, project_name, target_model) do |update_action|
-                    puts "Executing #{update_action.raw}..."
+                    puts "Executing #{update_action.to_h}..."
                   end
                   puts "Success!"
                 end
 
                 return
               else
-                puts "To commit, run \033[1;31m#{program_name} #{command_name} --file #{file} --target-model #{target_model} --execute\033[0m"
+                puts "To commit, run \033[1;31m#{program_name} #{project_name} --file #{file} --target-model #{target_model} --execute\033[0m"
```

Comment:
This was duplicating the `add` command in the instructions. I think `program_name` already includes `add`.

Reply:
I already have this fixed in my latest branch actually, but in a different way, can we revert?  Or I can just remove when I merge y branch in.

Reply:
Actually don't worry about it, I'll just address on my merge.

Reply:
Okay, that sounds good.